backup-unit/browserunit.cpp

```diff
@@ -0,0 +1,105 @@
+#include <vault/unit.h>
+#include <qtaround/os.hpp>
+
+#include <QProcess>
+#include <QCoreApplication>
+#include <QStringList>
+#include <QVariantList>
+#include <QVariantMap>
+#include <QDebug>
+#include <sys/types.h>
+#include <signal.h>
+#include <set>
+#include <unistd.h>
+#include <stdexcept>
+
+
+void stop_browser()
+{
+    qDebug() << "Terminating browser";
+    QProcess ps;
+    auto get_browser_pids = [&ps]() {
+        std::set<int> res;
+        ps.start("pgrep", {"-f", "sailfish-browser"});
+        ps.waitForFinished(-1);
+        if (ps.exitStatus() == QProcess::CrashExit) {
+            throw std::runtime_error("pgrep failed");
+            return res;
```

Comment:
Which one is more important? I see that there's try-catch in the main. The get_browser_pids of set<int> won't have anything meaningful as this return won't be handled.

I think qCWarning + return res as empty should sufficient.